internal/app/configurable.go

```diff
@@ -80,6 +81,7 @@ const (
 	BatchByTimeAndCount = "bytimecount"
 	IsEventData         = "iseventdata"
 	MergeOnSend         = "mergeonsend"
+	HttpRequestHeaders  = "httpRequestHeaders"
```

Comment:
Value are all lower case. This is because there is code the converts the incoming to lower case for matching purposes.
```suggestion
	HttpRequestHeaders  = "httprequestheaders"
```